packages/jam/node/export.ts

```diff
@@ -0,0 +1,75 @@
+import fs from "node:fs";
+import path from "node:path";
+import type { Block } from "@typeberry/block";
+import { Block as BlockCodec } from "@typeberry/block";
+import { Bytes } from "@typeberry/bytes";
+import { Encoder } from "@typeberry/codec";
+import { LmdbBlocks } from "@typeberry/database-lmdb";
+import { Blake2b } from "@typeberry/hash";
+import { Logger } from "@typeberry/logger";
+import { getChainSpec, openDatabase } from "./common.js";
+import type { JamConfig } from "./jam-config.js";
+
+export async function exportBlocks(jamNodeConfig: JamConfig, outputDir: string, withRelPath: (p: string) => string) {
+  const logger = Logger.new(import.meta.filename, "export");
+
+  logger.info`📤 Exporting blocks to ${path.resolve(outputDir)}`;
+
+  if (!fs.existsSync(outputDir)) {
+    fs.mkdirSync(outputDir, { recursive: true });
+  }
```

Comment:
Export to single file could be added here, by detecting:
1. If the destination exists and is directory = export separate files.
2. If the destination does not exist (or is a file) = open the file (reset content) and append blocks there.

That said, it's absolutely fine to do as follow up. @coderabbitai please create an issue for this.